lib/ansible/modules/cron.py

```diff
@@ -612,6 +612,7 @@ def main():
     res_args = dict()
     warnings = list()
 
+try:
```

Comment:
Since this change is unrelated to the new module, it looks like it was included unintentionally. It's also what's responsible for causing most of the CI failures, since it moves a section of the `main` function into the global scope.

Reverting this change (and the `except` below) will remove most of the test failures -- although as was already mentioned, the new module in this PR is something that would be better in a collection than in core.

Reply:
I think it might have been an inadvertent change during rebasing on 2.16. The PR should be rebased on `devel` and target `devel` (only bugfixes are backported to stable branches, after being merged to devel).

@Naya1217 New features are added very selectively to ansible-core at this point, but I hope this doesn't discourage you, because developing a module and making a PR that passes CI is great practice with git and ansible. Let me know if you have any questions about my feedback, I'd  be happy to clarify or hop on a call to do some pair programming if you want.